### PR TITLE
chore: update .dockerignore

### LINKED ISSUE #447: update dockerignore file
check what can be added to the dockerignore file. Specifically the infra folder

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- 

## Context

- fixes #447 

## Checklist

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [x] Make sure that all checks pass

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -3,3 +3,27 @@ node_modules
...
+localenv
+
+.github
+.husky
+.dockerignore
+.eslintignore
+.eslintrc.yml
+.gitattributes
+.gitignore
+.lintstagedrc.js
```
Comment (raducristianpopa) [inline .dockerignore:20]: I think `.lintstagedrc.js` can be deleted, since it's not used anymore.

File: packages/frontend/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:18.13.0-slim as builder
```
Comment (mkurapov) [inline packages/frontend/Dockerfile:1]: :+1:

File: .dockerignore [hunk 3]
```diff
@@ -3,3 +3,26 @@ node_modules
...
+.gitignore
+.prettierignore
+commitlint.config.js
+jest.config.base.js
+renovate.json
+
+**/jest.config.js
+**/jest.setup.js
+**/jest.teardown.js
+**/docker-compose.yml
```
Comment (mkurapov) [inline .dockerignore:28]: maybe we can also ignore test files in here?

File: .dockerignore [hunk 4]
```diff
@@ -3,3 +3,26 @@ node_modules
 **/node_modules
 **/dist
 dist
+tmp
+**/tmp
```
Comment (mkurapov) [inline .dockerignore:7]:
```suggestion
```
I think `tmp` already covers this

File: .dockerignore [hunk 5]
```diff
@@ -3,3 +3,26 @@ node_modules
 **/node_modules
 **/dist
```
Comment (mkurapov) [inline .dockerignore:4]:
```suggestion
```
same comment as below

File: .dockerignore (hunk 3 above)
Comment (sabineschaller) [reply .dockerignore:28]: Yes!

File: .dockerignore (hunk 4 above)
Comment (sabineschaller) [reply .dockerignore:7]: I have tmp folders in the root and in sub-directories.

Comment (sabineschaller) [reply .dockerignore:3]: Never mind, I read up on the matter.

Comment (mkurapov) [reply .dockerignore:3]: I think just `tmp` would also cover root and sub-directories, but `**/tmp` works as well

**Review by mkurapov — APPROVED** [review]

File: .dockerignore (hunk 4 above)
Comment (sabineschaller) [reply .dockerignore:3]: From what I read, `tmp` only covers the root directory but `**/tmp` covers root and sub-directories. But I did find conflicting information.

**Pull request merged**